kolibri/core/assets/src/core-app/import-intl-locale.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (locale, callback) => {
+  let promise;
+  switch (locale) {
+    case 'sw-TZ':
+      promise = new Promise((resolve) => {
+        require.ensure([], (require) => {
+          resolve(() => require('intl/locale-data/jsonp/sw-TZ.js'));
+        });
+      });
+      break;
+    case 'es-ES':
+      promise = new Promise((resolve) => {
+        require.ensure([], (require) => {
+          resolve(() => require('intl/locale-data/jsonp/es-ES.js'));
+        });
+      });
+      break;
+    default:
+      promise = new Promise((resolve) => {
+        require.ensure([], (require) => {
+          resolve(() => require('intl/locale-data/jsonp/en.js'));
+        });
+      });
+      break;
+  }
+  return promise;
+};
```

Comment:
Yeah, would be better. I don't think our (soon to be linter) is completely averse to returns in switch cases, just empty returns being used in place of breaks. Will change it to returns, and modify in the new linting PR if it complains.